example_sp/sp-repoze/attributemaps/basic.py

```diff
@@ -0,0 +1,326 @@
+
+MAP = {
+    "identifier": "urn:oasis:names:tc:SAML:2.0:attrname-format:basic",
+    "fro": {
+        'urn:mace:dir:attribute-def:aRecord': 'aRecord',
```

Comment:
That's why you only get `sn` as attribute and not all the attributes.
Interesting, that could be `'urn:oasis:names:tc:SAML:2.0:attrname-format:uri'` instead. We have to check, I will.